scripts/variantreport_p_sample.rmd

```diff
@@ -330,26 +338,26 @@ DT::datatable(NoSignatureTable,
 
 ```{r deconvolution, message=F, warning=F, include = FALSE}
 # markdown text:
-## Deconvolution
-### Deconvolution  
+# # Deconvolution
+# ## Deconvolution  
 # Deconvolution is done by using robust regression. The resulting coefficients are normalized to get the percentage of 
 # each present variant. 
-
+# 
 # prepare unique signature mutation dataframe
-
+# 
 # sig_mutations_unique.df <- sig_mutations.df 
 # unique_stuff <- unique(sig_mutations.df$value)
 # sig_mutations_unique.df <- subset(sig_mutations.df, !duplicated(value))
 # sig_mutations_unique.df <- sig_mutations_unique.df %>%
 #                                 tidyr::pivot_wider(values_fn = unlist)
-
+# 
 # 1. count occurences of mutation
 
 ```
 ```{r deconvolution_prep, message=F, warning=F, include = FALSE}
 
 # filter match table for unique mutations
-match_unique.df <- dplyr::distinct(match.df[,-13])
+match_unique.df <- unique(dplyr::distinct(match.df[,-13]))
```

Comment:
I don't like the magic -13 here.  Can this get a proper name and possibly even be derived from file headers?